AngularSpecificAttributesRemover.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const jestDOMElementSerializer = require('pretty-format').plugins.DOMElement;
+
+const serialize = (node, ...rest) => {
+  const nodeCopy = node.cloneNode(true);
+  Object.values(nodeCopy.attributes)
+    .map(attribute => attribute.name)
+    .filter(attributeName => attributeName.includes('ng-reflect'))
+    .forEach(attributeNameToRemove => nodeCopy.attributes.removeNamedItem(attributeNameToRemove));
+
+  return jestDOMElementSerializer.serialize(nodeCopy, ...rest);
+};
+
+const test = val =>
+  jestDOMElementSerializer.test(val) && serializeTestFn(val);
+
+module.exports = {
+  serialize: serialize,
+  test: test
+};
+
+function serializeTestFn(val) {
+  return val.attributes !== undefined && Object.values(val.attributes)
+    .map(attribute => attribute.name)
+    .some(attributeName => attributeName.includes('ng-reflect'))
```

Comment:
I think `attributeName.startsWith` is more appropriate here, Angular's naming convention is that Angular specific attributes start with `ng`. Imagine someone having an attribute called `some-wrong-reflect-attribute`.

Reply:
Done